### DISCUSSION TITLE
the question about kidney PK after iv bolus

### DISCUSSION THREAD

**Discussion by Carrie-nothing** [Open-Systems-Pharmacology/Forum#1404] [Questions & Problems]
Hello teams,
I have one question about the kidney PK after iv bolus.
In my opinion, the PK of kidney should show the accumulation distribution curve rather than the elimination curve of whole blood. The results of the software simulation are as follows:
![138796cfafffd27bee67276fa731917](https://user-images.githubusercontent.com/92715703/206603474-7a69956b-2e7f-4159-a261-efeed6b9d611.jpg)

The results I think should be shown in the following figure:
![9de637c7dce34b90a685e7fa32dc7b4](https://user-images.githubusercontent.com/92715703/206603498-037ef478-fc48-4bc6-ab08-fe40ae2efb42.jpg)

So I really hope the team can give me this answer, it will be a great help!

Looking forward to your reply, thank you very much!
                                                                          Yours,
                                                                           Carrie.

**Comment by tobiasK2001** [comment]
Dear Carrie,

just on a quick note: why do you expect an accumulation? Is your compound TAC so similar to VAN? Do you have tissue measurements for TAC in kidney or any other hint?
Best, Tobias
